lib/codemod/src/transforms/__testfixtures__/convert-to-module-format/multi.output.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+import React from 'react';
+import Button from './Button';
+
+export default {
+  title: 'Button1',
+};
+
+export const story1 = () => <Button label="Button1.1" />;
+export const story2 = () => <Button label="Button1.2" />;
+
+export default {
```

Comment:
This is invalid code, an ES module cannot have the same export twice

Reply:
Do a detection, then either
- throw
- skip this file

Then warn user

Reply:
I'm outputting a warning but choose to output invalid code, as documented in the README:

```
- If a file has multiple `storiesOf` declarations, it will convert each one seperately. This generates invalid ES6, but you can edit the file by hand to split it into multiple files (or whatever is appropriate).
```